recipe/meta.yaml

```diff
@@ -35,6 +35,9 @@ test:
     - dask
   commands:
     - pip check
+    - dask docs --help
+    - dask info --help
+    - dask info versions --help
```

Comment:
Tests are failing because the CLI requires `click`; are we okay with adding `click` as a core dependency for the Dask conda packages? Or are these tests better suited for the `dask` metapackage

Reply:
Yes think that is reasonable

We should then also added it to [Dask's `setup.py`]( https://github.com/dask/dask/blob/4a350fceed75d4d461366ec8b44e66955f83b5d9/setup.py#L35-L42 ) and [the nightly package]( https://github.com/dask/dask/blob/4a350fceed75d4d461366ec8b44e66955f83b5d9/continuous_integration/recipe/meta.yaml#L25 )

Would reuse Distributed [`click >=6.6` requirement]( https://github.com/dask/distributed/blob/09837312ccb3a9f1a14ea068ba5825963fec82cc/requirements.txt#L1 )

Reply:
I initially suggested adding `click` as a required dependency of `dask` (pip) / `dask-core` (conda) but there was some push back against that. I personally don't have strong thoughts one way or the other. I agree we should have these. We currently have [these types of checks over in the `dask-feedstock`](https://github.com/conda-forge/dask-feedstock/blob/fa81d16d6350f91f0b6b042c3a17da2612517973/recipe/meta.yaml#L49-L50) due to the current places that `click` is installed

Reply:
Yeah think we still want some kind of check in this feedstock to make sure the entry point added here works

We could just add click to `test/requires` as is done with `pip` below

That said, I think not including `click` as a requirement in Dask itself while adding entry points is a subpar user experience. Think we should bring this up again and point back to this issue (and its resolution) as a reason to change it